helm/dagster/schema/schema/utils/helm_template.py

```diff
@@ -77,7 +77,7 @@ def render(
             k8s_objects = [k8s_object for k8s_object in yaml.full_load_all(templates) if k8s_object]
             if self.model:
                 k8s_objects = [
-                    self.api_client._ApiClient__deserialize_model(  # pylint: disable=W0212
+                    self.api_client._ApiClient__deserialize_model(  # pylint: disable=W0212  # noqa: SLF001
```

Comment:
Should we get rid of the pylint disable here? A bunch of other instances of this as well.

Reply:
Behold upstack: https://github.com/dagster-io/dagster/pull/12162